addons/hr_expense/models/hr_expense.py

```diff
@@ -353,7 +353,7 @@ def create_expense_from_attachments(self, attachment_ids=None, view_type='list')
 
         for attachment in attachments:
             expense = self.env['hr.expense'].create({
-                'name': product.display_name,
```

Comment:
Spec is a bit too much to find it back exactly - but checking back the comments for the original PR -  
`set Description = product, for the expenses that are created by/from scanning`

But again, I would say up to a current PO to decide whether it's proper or not ^^

Reply:
But at that point the product is the same for all attachments :confused:

Reply:
true true, if I recall correctly, when creating expense from attachment, it picks General product or something like that.. So the change makes a lot of sense

Reply:
If then name got changed after the scan, if the scan detects a product, it would make sense to name it as the product though; not an expert in Expense so not sure :sweat_smile: